seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/backend/BackendClient.java

```diff
@@ -0,0 +1,264 @@
+                ex = e;
+            }
+        }
+        log.error(ErrorMessages.CONNECT_FAILED_MESSAGE, routing);
+        //        throw new ConnectedFailedException(routing.toString(), ex);
+        throw new DorisConnectorException(
+                DorisConnectorErrorCode.SCAN_BATCH_FAILED, routing.toString(), ex);
+    }
+
+    /**
+     * get next row batch from Doris BE
+     *
+     * @param nextBatchParams thrift struct to required by request
+     * @return scan batch result
+     * @throws DorisConnectorException throw if cannot connect to Doris BE
+     */
+    public TScanBatchResult getNext(TScanNextBatchParams nextBatchParams) {
+        log.debug("GetNext to '{}', parameter is '{}'.", routing, nextBatchParams);
+        if (!isConnected) {
+            open();
+        }
+        TException ex = null;
+        TScanBatchResult result = null;
+        for (int attempt = 0; attempt < retries; ++attempt) {
+            log.debug("Attempt {} to getNext {}.", attempt, routing);
+            try {
+                result = client.getNext(nextBatchParams);
+                if (result == null) {
+                    log.warn("GetNext result from {} is null.", routing);
+                    continue;
+                }
+                if (!TStatusCode.OK.equals(result.getStatus().getStatusCode())) {
+                    log.warn(
+                            "The status of get next result from {} is '{}', error message is: {}.",
+                            routing,
+                            result.getStatus().getStatusCode(),
+                            result.getStatus().getErrorMsgs());
+                    continue;
+                }
+                return result;
+            } catch (TException e) {
+                log.warn("Get next from {} failed.", routing, e);
+                ex = e;
+            }
+        }
+        if (result != null && (TStatusCode.OK != (result.getStatus().getStatusCode()))) {
+            log.error(
+                    ErrorMessages.DORIS_INTERNAL_FAIL_MESSAGE,
+                    routing,
+                    result.getStatus().getStatusCode(),
+                    result.getStatus().getErrorMsgs());
+            //            throw new DorisInternalException(routing.toString(),
+            // result.getStatus().getStatusCode(),
+            //                    result.getStatus().getErrorMsgs());
+            String errMsg =
+                    "Doris server "
+                            + routing.toString()
+                            + " internal failed, status code ["
+                            + result.getStatus().getStatusCode()
+                            + "] error message is "
+                            + result.getStatus().getErrorMsgs();
+            throw new DorisConnectorException(DorisConnectorErrorCode.SCAN_BATCH_FAILED, errMsg);
+        }
+        log.error(ErrorMessages.CONNECT_FAILED_MESSAGE, routing);
```

Comment:
Why log `ErrorMessages.CONNECT_FAILED_MESSAGE` here? If the code runs here, does it mean there is no problem with the connection?